src/explore-education-statistics-admin/src/components/editable/EditableAccordionSection.tsx

```diff
@@ -149,25 +154,46 @@ const EditableAccordionSection = (props: EditableAccordionSectionProps) => {
                   {isEditingHeading ? (
                     <Button onClick={saveHeading}>Save section title</Button>
                   ) : (
-                    <Button
-                      type="button"
-                      onClick={toggleEditingHeading}
-                      variant="secondary"
+                    <Tooltip
+                      text={disabledHeadingChangeTooltip}
+                      enabled={!!disabledHeadingChangeTooltip}
                     >
-                      Edit section title
-                    </Button>
+                      {({ ref }) => (
+                        <Button
+                          ariaDisabled={!!disabledHeadingChangeTooltip}
+                          type="button"
+                          ref={ref}
+                          variant="secondary"
+                          onClick={toggleEditingHeading}
+                        >
+                          Edit section title
+                        </Button>
+                      )}
+                    </Tooltip>
                   )}
 
                   {headerButtons}
 
                   {onRemoveSection && (
                     <>
-                      <Button onClick={toggleRemoveModal.on} variant="warning">
-                        Remove this section
-                      </Button>
+                      <Tooltip
+                        text={disabledRemoveSectionTooltip}
+                        enabled={!!disabledRemoveSectionTooltip}
+                      >
+                        {({ ref }) => (
+                          <Button
+                            ariaDisabled={!!disabledRemoveSectionTooltip}
+                            ref={ref}
+                            variant="warning"
+                            onClick={toggleRemoveModal.on}
+                          >
+                            Remove this section
+                          </Button>
+                        )}
+                      </Tooltip>
```

Comment:
I originally wrote this pattern into the `Button` component itself, but decided against it as it would lead to tooltip code being unnecessarily included in bundles for the public frontend (where tooltips are unlikely to be used).

We've also discussed this before, but tooltips should probably be avoided so it didn't seem like a good idea to cement the pattern into a component on reflection. 

This PR's usages of tooltips is a little on the niche side as well, so it'd be a good idea to  other parts of the UI begin to require the use of tooltips.